Automattic/wp-calypso
# Onboaring: add blogger flow

## Changes proposed in this Pull Request

This migrates the current blogger/write flow from the signup to the new stepper framework.

<img width="1890" alt="Markup 2022-03-18 at 16 03 44" src="https://user-images.githubusercontent.com/33258733/159029764-3e5d20da-cb0e-4dcd-ba9f-9d04801b423f.png">


## Testing instructions

* Checkout the branch
* Run `yarn start`
* Check that the blogger flow is working
